app/build.gradle

```diff
@@ -12,10 +12,11 @@ android {
 
     defaultConfig {
         applicationId "org.openobservatory.ooniprobe"
-        minSdkVersion 14
+        minSdkVersion 23
```

Comment:
Sure, I just wanted to be sure, thanks!

We should probably use 14, which is the minimum for which FCM is available and for which the current NDK allows to cross compile. Excluding two API levels does not seem to me to give us any specific gain, notwithstanding the user distribution (which I also know).